services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouteEntryController.java

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.route.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ComponentScan(value = "com.futurewei.alcor.common.stats")
+public class RouteEntryController {
```

Comment:
btw, doesn't find APIs in RouteEntryController and RouteTableController. Do we still need these new controllers?

Reply:
RouteEntryController is removed and RouteTableController is empty